### DISCUSSION TITLE
RgbwColor - Get the individual elements

### DISCUSSION THREAD

**Discussion by FillDee** [Makuna/NeoPixelBus#547] [Q&A] [answered]
Is there any way to get the individual RGB and W values from an RgbwColor object?
Something like color[R], or color[0] etc.
Say I passed an RgbwColor to a function and I wanted to alter that colour in some way. EG, reduce the blue value by half at night time Any way to do that? 
I know I could pass the reduced values in the first place, but that's not what I'm looking to do.
Thanks in advance.

**Comment by Makuna** [comment, marked as answer]
The variables are exposed publicly.  [Here is the Wiki entry for the API for RgbColor](https://github.com/Makuna/NeoPixelBus/wiki/RgbColor-object-API)

color.R, color.G, color.B, and color.W

**Reply by FillDee** [reply]
Wow. How stupid do I feel now....
Many thanks for the swift answer and the library.
